### ISSUE TITLE
Toolbar hidden behind image

### ISSUE THREAD

**Issue by ellatrix** [WordPress/gutenberg#16529]
**Describe the bug**

<img width="452" alt="Screenshot 2019-07-08 at 15 43 42" src="https://user-images.githubusercontent.com/4710635/60815113-3dc58800-a197-11e9-9497-ede1ec4c6777.png">

The toolbar is now hidden behind the previous block. This is caused by `translate3d(0px, 0px, 0px)` after #16065.

**To reproduce**
See demo content, full size image.

**Comment by ellatrix** [comment]
Oops, duplicate of #16519.
